bin/webpack-helpers.js

```diff
@@ -123,6 +125,9 @@ const mainEntry = {
 	'active-filters': './assets/js/blocks/active-filters/index.js',
 	'block-error-boundary':
 		'./assets/js/base/components/block-error-boundary/style.scss',
+
+	// cart & checkout blocks
```

Comment:
Cart & Checkout blocks should be treated as separate blocks, both in folder and naming, I would suggest we have a top-level block folder for each one of them.

Reply:
That was my thinking - our blocks folder is getting hard to navigate, and naming is hard. Folders for the main groups can help break it up :)